apps/client/package.json

```diff
@@ -4,7 +4,7 @@
   "scripts": {
     "dev": "yarn dev-codegen & yarn dev-next",
     "dev-codegen": "graphql-codegen --watch",
-    "dev-next": "next dev -p 3010",
+    "dev-next": "next dev -p 3010 --turbo",
```

Comment:
are we using turbo here?

Reply:
yes, but I will leave this setting change for Norris!